src/StaticAnalysis/CacheWarmer.php

```diff
@@ -29,8 +29,16 @@ public function warmCache(string $cacheDirectory, bool $useAnnotationsForIgnorin
         );
 
         foreach ($filter->files() as $file) {
+            /* @noinspection UnusedFunctionResultInspection */
+            $coveredFileAnalyser->classesIn($file);
```

Comment:
These are needed too because cache are per-method:

https://github.com/sebastianbergmann/php-code-coverage/blob/9871cf421031c8a103e246b0bcf460a317d95174/src/StaticAnalysis/Cache.php#L85-L88